incubator/spark-history-server/charts/nfs/values.yaml

```diff
@@ -0,0 +1,11 @@
+namespace: spark-history-server
+
+gceStorage: 5Gi
```

Comment:
What is `gceStorage` why is it alongside the other PVC?

Reply:
There are two required PVCs, one for the NFS server front-end and another for the server itself. The sub-chart is adapted from this [example](https://github.com/kubernetes/examples/tree/master/staging/volumes/nfs), where there is some documentation.